.jsdoc.json

```diff
@@ -1,16 +1,24 @@
 {
   "source": {
-    "include": [ "src/js/" ],
+    "include": ["src/js/"],
     "includePattern": ".js$"
   },
   "opts": {
     "destination": "docs/api",
     "readme": "docs/index.md",
-    "template": "node_modules/tui-jsdoc-template",
-    "package": "package.json",
+    "template": "node_modules/clean-jsdoc-theme",
+    "package": "",
```

Comment:
with this theme, this keeps the output in the root of the destination dir and not at package/version/